README.md

```diff
@@ -48,6 +48,10 @@ https://www.terraform-best-practices.com/
 State is stored in S3, and will be picked up automatically from the init command.
 It is backend "s3" section of the main.tf file.
 
+## Running Terraform in sandbox environment
+
+Resource naming in the sandbox environment (AWS Account 563181399728) has been changed to include the user's CRSid. When running Terraform commands, you will be prompted to enter a value for the `owner` for which the CRSid should be provided. This will be added as a prefix in resource names. Other environments, dev, staging and production 
+
```

Comment:
I've updated the note with your suggested wording. Currently it prompts in all environments because the Owner tag is added in all environments. It could be made optional however

Reply:
I don't think we want the owner tag in the other environments, just in sandbox as we want to merge and deploy changes to one fixed deployment in dev, staging and live.

Reply:
Okay I'll get rid of that in the other envs 👍